db/seeds/feeds.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+unless Feed.find_by_slug('fashion-tumblr').present?
+  Feed.create!(
+    name: 'Fashion Tumblr',
+    url: 'http://europeanafashion.tumblr.com/rss'
+  )
+
+  # if the landing page is already present we need to add our feed to it here
```

Comment:
I was trying to catch all scenarios in terms of (re)running combinations of migration and seed tasks. Considering seed should only be ran once, You're right though, just moving this to the migration is the best option.